Sources/generate-swift-syntax-builder/gyb_syntax_support/AttributeKinds.py

```diff
@@ -0,0 +1,871 @@
+
+# True if this shouldn't be serialized.
+NotSerialized = 'NotSerialized'
+
+# True if this attribute is only valid when parsing a .sil file.
+SILOnly = 'SILOnly'
+
+# The attribute should be reported by parser as unknown.
+RejectByParser = 'RejectByParser'
+
+# Whether client code cannot use the attribute.
+UserInaccessible = 'UserInaccessible'
+
+# Whether adding this attribute can break API
+APIBreakingToAdd = 'APIBreakingToAdd'
+
+# Whether removing this attribute can break API
+APIBreakingToRemove = 'APIBreakingToRemove'
+
+# Whether adding this attribute can break ABI
+ABIBreakingToAdd = 'ABIBreakingToAdd'
+
+# Whether removing this attribute can break ABI
+ABIBreakingToRemove = 'ABIBreakingToRemove'
+
+# The opposite of APIBreakingToAdd
+APIStableToAdd = 'APIStableToAdd'
+
+# The opposite of APIBreakingToRemove
+APIStableToRemove = 'APIStableToRemove'
+
+# The opposite of ABIBreakingToAdd
+ABIStableToAdd = 'ABIStableToAdd'
+
+# The opposite of ABIBreakingToRemove
+ABIStableToRemove = 'ABIStableToRemove'
+
+# Whether this attribute is only valid when concurrency is enabled.
+ConcurrencyOnly = 'ConcurrencyOnly'
+
+# Whether this attribute is valid on additional decls in ClangImporter.
+OnAnyClangDecl = 'OnAnyClangDecl'
+
+TYPE_ATTR_KINDS = [
+  # Type attributes
+  TypeAttribute('autoclosure'),
+  TypeAttribute('convention'),
+  TypeAttribute('noescape'),
+  TypeAttribute('escaping'),
+  TypeAttribute('differentiable'),
+  TypeAttribute('noDerivative'),
+  TypeAttribute('async'),
+  TypeAttribute('Sendable'),
+  TypeAttribute('unchecked'),
+  TypeAttribute('_typeSequence'),
+  TypeAttribute('_local'),
+  TypeAttribute('tuple'),
+
+  # Generated interface attributes
+  TypeAttribute('_opaqueReturnTypeOf'),
+]
+
+# Schema for DECL_ATTR:
+#
+# - Attribute name.
+# - Class name without the 'Attr' suffix (ignored for
+# - Options for the attribute, including:
+#    * the declarations the attribute can appear on
+#    * whether duplicates are allowed
+#    * whether the attribute is considered a decl modifier or not (no '@')
+# - Unique attribute identifier used for serialization.  This
+#   can never be changed.
+#
+# SIMPLE_DECL_ATTR is the same, but the class becomes
+# SimpleDeclAttr<DAK_##NAME>.
+#
+# Please help ease code review/audits:
+# - Please indent once, not to the opening '('.
+# - Please place the "OnXYZ" flags together on the next line.
+# - Please place the non-OnXYZ flags together on the next to last line.
+# - Please place the unique number on the last line. If the attribute is NOT
+#   serialized, then please place that flag on the last line too. For example:
+#     code=123),
+#     NotSerialized, code=321),
+# - Please sort attributes by serialization number.
+# - Please create a "NOTE" comment if a unique number is skipped.
+#
+# If you're adding a new underscored attribute here, please document it in
+# docs/ReferenceGuides/UnderscoredAttributes.md.
+DECL_ATTR_KINDS = [
+  DeclAttribute('_silgen_name', 'SILGenName',
```

Comment:
Nit: We use four space for indentation in Python so all all the members of `DECL_ATTR_KINDS` should be indented by four spaces instead of two.

(same for `TYPE_ATTR_KINDS`)